base/ca/src/main/java/com/netscape/ca/CASigningUnit.java

```diff
@@ -55,11 +55,11 @@ public CASigningUnit() {
 
     @Override
     public void updateConfig(String nickname, String tokenname) {
```

Comment:
All the `updateConfig` in the subclasses call the same method. Is there a reason to not implement in the super class?